packages/zoe/package.json

```diff
@@ -11,7 +11,7 @@
     "build": "yarn build:bundles",
     "build:bundles": "node scripts/build-bundles.js",
     "prepack": "echo \"export {}; \" | cat - tools/types-ambient.js > tools/types.js && tsc --build jsconfig.build.json",
-    "postpack": "git clean -f '*.d.ts*'",
+    "postpack": "git clean -f '*.d.ts*' tools/types.js",
```

Comment:
this is more correct. was it also part of an observed problem?

Reply:
Yes, leaving behind garbage in the repo after `yarn pack`.